src/hatti/views/table.cljs

```diff
@@ -121,14 +120,19 @@
   ([form get-label? language & {:keys [is-filtered-dataview? owner]}]
    (let [columns (for [field (all-fields form :is-filtered-dataview?
                                          is-filtered-dataview?)]
-                   (let [{:keys [name type full-name]} field
+                   (let [{:keys [name type full-name], {:keys [hxl]} :instance} field
                          label (if get-label? (get-label field language) name)]
-                     {:id name :field full-name :type type
-                      :name label :toolTip label :sortable true
+                     {:id name
+                      :field full-name
+                      :type type
+                      :name (if hxl (str label "<div class=\"hxl-row\">" hxl "</div>") label)
```

Comment:
Makes sense, I'm not sure what the answer is, I assumed because it was a 
coding row. But in a hypothetical scenario with an OpenMRS concept 
coding row, would we want that to appear differently? If so, then I 
agree that keeping it as is makes sense.